reports/sig-security/2017-05-24.md

```diff
@@ -26,3 +26,61 @@ Announcement: [Moby project forum post](https://forums.mobyproject.org/t/introdu
   - we can propose additional deep dives and discussion topics!
 
 ## Meeting Notes
+
+* Administrivia
+  * There is a code of conduct
+  * Attendees from Docker, Intel, HPE, Google, IBM, ARM, Arkxan Technologies
+* What is LinuxKit?
+  * LinuxKit is a toolkit for building container-focused Linuxen. i.e. distro
+    building tool, not a distro itself
+  * Grew out of Docker for \* ({AWS, Mac, etc.})
+  * Borrowed userspace mostly from Alpine
+  * system daemons (e.g. DHCP, possibly SSH, etc.) run in containers, which are
+    distributed as Docker images
+  * base OS is immutable, since daemons are containers
+* Projects
+  * [Clear Containers](../../projects/clear-containers/)
+    * Question: what's the Intel feeling r.e. kvmtool, are they still
+      interested in using it for clear containers?
```

Comment:
To add some context here... `qemu` is big and bloated and has a scary attack surface. `kvmtool` promised to be a super minimal hypervisor, only emulating virtio devices, to reduce attack surface and also increase boot time considerably. It looked like Intel clear containers initially went this route, but then they switched gears to pursue some kind of qemu-integrated-orchestration monstrosity. I have no idea what became of all of this, but I'd be quite happy to learn that `kvmtool` will continue to have resources poured into keeping it lean, minimal, functional, and secure.